multiapps-controller-process/src/main/java/org/cloudfoundry/multiapps/controller/process/util/OperationInFinalStateHandler.java

```diff
@@ -115,13 +116,13 @@ private void trackOperationDuration(String correlationId, DelegateExecution exec
         ProcessTime overallProcessTime = operationTimeAggregator.computeOverallProcessTime(correlationId, processTimes);
         
         DynatraceProcessDuration dynatraceProcessDuration = ImmutableDynatraceProcessDuration.builder()
```

Comment:
It would be good to have a quick check that the new field in Operation object will not brake dynatrace metrics for operation events

Reply:
Done. It will not affect the Dynatrace objects.

Reply:
Also it is possible to require some changes in uptime monitoring system because there is assert logic for operation state